linux/opt/sage/bin/apache_conf_rewrite.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+. /opt/sage/bin/instance_env_vars.sh  && sed -i "s/EC2_INSTANCE_ID/$EC2_INSTANCE_ID/g" /etc/apache2/sites-available/proxy.conf
```

Comment:
it's good practice to not run a bash script within another bash script.  how about source it from the CNF template, like so..   https://github.com/Sage-Bionetworks/service-catalog-library/blob/master/ec2/sc-ec2-linux-jumpcloud.yaml#L318

Reply:
This PR (#10) doesn't run a shell script, it sources the environment variables file (which we've named instance_env_vars.sh) and just runs sed across an Apache conf. The Apache config file is embedded in an AMI through this PR to rstudio-packer: https://github.com/Sage-Bionetworks-IT/packer-rstudio/pull/40

The script in this PR (#10) is run via the user info statement in the notebook EC2 template for the notebooks product, which I PR here: https://github.com/Sage-Bionetworks/service-catalog-library/pull/213